components/service/experiments/src/main/java/mozilla/components/service/experiments/debug/ExperimentsDebugActivity.kt

```diff
@@ -61,6 +70,17 @@ class ExperimentsDebugActivity : Activity() {
             }
         }
 
+        if (intent.hasExtra(OVERRIDE_CLEAR_ALL_EXTRA_KEY)) {
+            clearJob = GlobalScope.launch(Dispatchers.IO) {
```

Comment:
I think it would be useful to have `logger.info` here describing what's being done (unless the methods being called here already do that -- I didn't check)...